proto/qdrant/cloud/iam/v1/iam.proto

```diff
@@ -498,3 +511,15 @@ enum UserConsentStatus {
   // period of time, we auto accept pending consents depending on the document type.
   USER_CONSENT_STATUS_PENDING = 3;
 }
+
+// buf:lint:ignore QDRANT_CLOUD_REQUIRED_REQUEST_FIELDS
+// LogoutRequest is the request for the Logout function
+message LogoutRequest {
+  // Empty
+}
+
+// LogoutResponse is the response from the Logout function
+message LogoutResponse {
+  // The actual user.
+  User user = 1;
+}
```

Comment:
Can you please move to https://github.com/qdrant/qdrant-cloud-public-api/pull/145/files#diff-dc2125f4452e7dfcdf10d7306b5fcae250fa91f4194793b9ab2b2617f6f19fcfR391

For details please see: https://github.com/qdrant/qdrant-cloud-public-api/blob/main/CONTRIBUTING.md 
```
Organize RPC methods in this order (list, get, create, update, delete, + additional methods). Their respective messages should follow the same structure for clarity.
```